config/checkstyle_non_main_files_checks.xml

```diff
@@ -23,6 +23,11 @@
                value="^(?!(.*href=|.*http(s)?:|import |(.* )?package |.* files=|.*\.dtd)).{101,}$"/>
       <property name="message" value="Line should not be longer than 100 symbols"/>
     </module>
+    <!--This check is to enforce -e for all maven commands -->
+    <module name="RegexpSingleline">
+        <property name="format" value="(mvn [^-])|(mvn \-[^e\-v])"/>
+        <property name="fileExtensions" value="sh, yml"/>
```

Comment:
please add `<property name="message" value="mvn execution should be with '-e' option to produce error message in case of problem"/>` in this case you do not need comment above, the comments the better.

https://books.sonatype.com/mvnref-book/reference/running-sect-options.html

Reply:
done~